var/spack/repos/builtin/packages/py-cmyt/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyCmyt(PythonPackage):
+    """Matplotlib colormaps from the yt project !"""
+    homepage = "https://yt-project.org"
+    url      = "https://github.com/yt-project/cmyt/archive/v1.0.4.tar.gz"
```

Comment:
```suggestion
    pypi = "cmyt/cmyt-1.0.4.tar.gz"
```
We prefer to download from PyPI instead of GitHub for the reasons outlined [here](https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github). Note that this will likely change the checksums. You can run `spack checksum py-cmyt` to compute the new checksums then copy-n-paste them.